src/main/test/java/ParserTest.java

```diff
@@ -0,0 +1,31 @@
+import Duke.Parser;
+import Duke.StringDatePair;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ParserTest {
+    @Test
+    public void inputEventTest(){
+        StringDatePair output = new Parser().parse(
```

Comment:
Indentation for wrapped lines should be 8 spaces (i.e. twice the normal indentation of 4 spaces) more than the parent line.